### ISSUE TITLE
Element template for Lanes and SequenceFlow

### ISSUE THREAD

**Issue by Lewik** [camunda/camunda-modeler#1389]
Version 3.1.0 fails to work with templates for Lanes and SequenceFlow.
Values of properties erased on save.

`<camunda:property name="label" value="thisWillBeRemovedOnSave" />`

Previously worked template (part):
```
  {
    "name": "Lane",
    "id": "bpmn.laneTemplate",
    "appliesTo": [
      "bpmn:Lane"
    ],
    "properties": [
      {
        "label": "User Group",
        "type": "String",
        "value": "",
        "binding": {
          "type": "camunda:property",
          "name": "userGroup"
        },
        "choices": [
        ]
      }
    ],
    "isDefault": true
  }
```

**Comment by nikku** [comment]
@philippfromme Is this a duplicate of https://github.com/camunda/camunda-modeler/issues/1373?

**Comment by Lewik** [comment]
Its very similar or the same.
This is not affects regular activities in my case. Only Lanes and SequenceFlows

**Comment by nikku** [comment]
Duplicate of https://github.com/camunda/camunda-modeler/issues/1373 (which has been fixed and released with `v3.1.2` of the app).
